### PR TITLE
added translations, added acf select option for post types

### PR DESCRIPTION
**Pull request by colesmedia**

### CODE REVIEW

File: source/php/App.php [hunk 1]
```diff
@@ -8,6 +8,15 @@ public function __construct()
     {
         add_action('wp_enqueue_scripts', array($this, 'enqueueStyles'));
         add_action('wp_enqueue_scripts', array($this, 'enqueueScripts'));
+
+        add_filter('acf/load_field/name=comments_from_post_type', array($this, 'acf_populate_select_options'));
+    }
+
```
Comment (sebastianthulin) [inline source/php/App.php]: Add dockblock.

File: source/php/Module/Comments/views/comments.blade.php [hunk 2]
```diff
@@ -2,34 +2,36 @@
...
         <div class="grid grid--columns" data-equal-container>
-            @foreach($comments as $comment)
-                <div class="u-flex grid-xs-12 grid-sm-12 grid-md-6 grid-lg-6">
-                    <a href="{{ get_comment_link($comment->comment_ID) }}" class="comment-card">
-                        <div class="comment-card-icon">
-                            <i class="icon-quote"></i>
-                        </div>
-
-                        
+            @if(count($comments) > 0)
```
Comment (sebastianthulin) [inline source/php/Module/Comments/views/comments.blade.php]: Use of the function empty() is faster.

File: source/php/Module/Comments/views/comments.blade.php [hunk 3]
```diff
@@ -2,34 +2,36 @@
...
+                                <div class="small-text">
+                                    <span class="author">
+                                        {{ get_user_meta($comment->user_id)['name_of_council_or_politician'][0] }}, 
+                                    </span>
+                                    <time class="date">
+                                        {{ explode(' ', $comment->comment_date)[0] }}
+                                    </time>
+                                </div>
+                                <p class="card-comment-content">
+                                    {{ substr($comment->comment_content, 0, 240) . '...' }}
```
Comment (sebastianthulin) [inline source/php/Module/Comments/views/comments.blade.php]: USe function wp_trim_words, place truncation in controller. Pass as variable.

**Review by sebastianthulin — CHANGES_REQUESTED** [review]

File: source/php/App.php (hunk 1 above)
Comment (colesmedia) [reply source/php/App.php]: Fixed.

File: source/php/Module/Comments/views/comments.blade.php (hunk 2 above)
Comment (colesmedia) [reply source/php/Module/Comments/views/comments.blade.php]: Fixed.

File: source/php/Module/Comments/views/comments.blade.php (hunk 3 above)
Comment (colesmedia) [reply source/php/Module/Comments/views/comments.blade.php]: Fixed.

**Pull request merged**